aws-rds-cfn-common/src/test/java/software/amazon/rds/common/handler/TaggingTest.java

```diff
@@ -168,13 +168,48 @@ void test_translateTagsToSdk() {
                 .resourceTags(resourceTags)
                 .build();
 
-        final Set<Tag> allTags = Tagging.translateTagsToSdk(tagSet);
+        final Collection<Tag> allTags = Tagging.translateTagsToSdk(tagSet);
 
         assertThat(allTags.containsAll(systemTags)).isTrue();
         assertThat(allTags.containsAll(stackTags)).isTrue();
         assertThat(allTags.containsAll(resourceTags)).isTrue();
     }
 
+    @Test
+    void test_translateTagsToSdk_duplicat_tags() {
+        String duplicateTagKey = "duplicate-tag-key-2";
+        String duplicateTagValue = "valid-value";
+        final Set<Tag> systemTags = Stream.of(
+                Tag.builder().key("system-tag-key-1").value("system-tag-value-1").build(),
+                Tag.builder().key("system-tag-key-2").value("system-tag-value-2").build()
+        ).collect(Collectors.toSet());
```

Comment:
I believe we need to collect it into a sorted set and ensure the expected order. It didn't matter much as RDS API will shuffle the tags anyways, but in the light of the presence of tag duplicates it sort of matters.

Reply:
Done.